dll/win32/devmgr/lang/pt-PT.rc

```diff
@@ -40,59 +40,59 @@ STRINGTABLE
-    IDS_DEV_LACKED_ARBITRATOR "O driver para este dispositivo requisitou um recurso que o ReactOS não sabe como manusear."
+    IDS_DEV_DEVLOADER_FAILED "O ReactOS não pôde carregar o controlador para este dispositivo porque o computador está a reportar dois tipos de barramento %1."
+    IDS_DEV_DEVLOADER_FAILED2 "O(s) carregadore(es) do dispositivo %1 não pôde/puderam carregar o controlador  para este dispositivo."
+    IDS_DEV_OUT_OF_MEMORY "O controlador para este dispositivo pode estar danificado, ou o sistema com pouca memória ou insuficiência de outros recursos."
+    IDS_DEV_ENTRY_IS_WRONG_TYPE "Este dispositivo não está a funcionar correctamente porque um de seus controladores pode estar danificado, ou o registo pode estar danificado."
+    IDS_DEV_LACKED_ARBITRATOR "O controlador para este dispositivo requisitou um recurso que o ReactOS não sabe como manusear."
     IDS_DEV_BOOT_CONFIG_CONFLICT "Outro dispositivo está a usar os recursos que este dispositivo necessita."
-    IDS_DEV_FAILED_FILTER "Os drivers para este dispositivo precisam ser reinstalados."
-    IDS_DEV_DEVLOADER_NOT_FOUND "Este dispositivo não está a funcionar correctamente porque o ReactOS não pôde carregar o arquivo %1 que carrega os drivers para o dispositivo."
-    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os drivers para este dispositivo está danificado."
-    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o driver para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
-    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorretamente."
-    IDS_DEV_INVALID_DATA2 "Este dispositivo não está a funcionar correctamente porque a BIOS do dispositivo está a reportar os recursos para este dispositivo incorretamente."
-    IDS_DEV_FAILED_START "Este dispositivo não está presente, não está a funcionar correctamente ou não tem todos os drivers instalados."
+    IDS_DEV_FAILED_FILTER "Os controladores para este dispositivo precisam ser reinstalados."
+    IDS_DEV_DEVLOADER_NOT_FOUND "Este dispositivo não está a funcionar correctamente porque o ReactOS não pôde carregar o arquivo %1 que carrega os controladores para o dispositivo."
+    IDS_DEV_DEVLOADER_NOT_FOUND2 "Este dispositivo não está a funcionar correctamente porque o arquivo %1 que carrega os controladores para este dispositivo está danificado."
+    IDS_DEV_DEVLOADER_NOT_FOUND3 "Falha no dispositivo: tente mudar o controlador para este dispositivo. Se isto não funcionar, consulte a documentação do seu hardware."
+    IDS_DEV_INVALID_DATA "Este dispositivo não está a funcionar correctamente porque a BIOS do seu computador está a reportar os recursos para este dispositivo incorrectamente."
```

Comment:
"incorrectamente" looks spanish? Isn't it "incorretamente" instead in portuguese?
also, "corretamente" ?

Reply:
Incorretamente is PT Brasilian, incorrectamente is PT  Portugal